docker-compose.yml

```diff
@@ -66,6 +80,10 @@ services:
       ALLU_USERNAME: ${ALLU_USERNAME}
       ALLU_PASSWORD: ${ALLU_PASSWORD}
       ALLU_INSECURE: ${ALLU_INSECURE}
+      ALLU_UPDATE_INITIAL_DELAY: ${ALLU_UPDATE_INITIAL_DELAY}
```

Comment:
WARN[0000] The "ALLU_UPDATE_INITIAL_DELAY" variable is not set. Defaulting to a blank string. 

I got that warning while running docker compose. Is that value supposed to be defined somewhere?

Reply:
Thank you for noticing. I had `ALLU_UPDATE_INITIAL_DELAY=60000000` in `.env`, so I didn't notice this.

I added an empty default value for this in docker-compose.yml to rid of the warning. I also added `ALLU_UPDATE_INITIAL_DELAY=60000` to .env.local as a hint to how to change the value.